src/asciidoc/docs/customizations/file.adoc

```diff
@@ -8,8 +8,7 @@
 //tag::description[]
 upload any file.
 //end::description[]
-* For specific file types, use the associated VCC where possible.
-For example, see <<image#>>, <<sound#>>, or <<video#>>.
+* Using a `file` VCC is equivalent to using a <<media#, media VCC>> with `acceptOnly` set to `["file"]`.
```

Comment:
The file type uploads a raw file (pdf, zip, etc.). If you use the file type to upload an image or video, it will not apply any transformations to the file - it will simply upload the original. The image/video/sound types will all transcode files or compress them to improve delivery, which may not be expected behavior in something like the Gumroad template, where I might be selling high-resolution photos for download, rather than for display in the browser.